docs/release-notes/release-notes-0.19.0.md

```diff
@@ -131,6 +131,9 @@ when running LND with an aux component injected (custom channels).
 * [Fix a bug](https://github.com/lightningnetwork/lnd/pull/9798) that could
   result in a new topology client missing a channel-close notifications. 
 
+* [Fix a bug](https://github.com/lightningnetwork/lnd/pull/9810) where the
```

Comment:
RC5 is about to drop, so this probably won't make it in. Can you move this to 0.19.1 please?